desk/lib/bolt.hoon

```diff
@@ -39,9 +38,7 @@
   +*  this  .
       ag    ~(. yosh bowl)
       io    ~(. agentio bowl)
-      perms  ~(. permissions bowl state)
-      allowed  %:(is-allowed:perms src.bowl)
-      emit   ~(website json state)
+      allowed  (is-allowed bowl)
```

Comment:
Since the only arm in `permissions` is `is-allowed`, which takes a value from the `bowl` as its only argument, I suggest collapsing these into one level, a gate called `is-allowed` which takes a bowl.